fe/fe-core/src/main/java/com/starrocks/transaction/PublishVersionDaemon.java

```diff
@@ -168,6 +169,18 @@ private void publishVersionForOlapTable(List<TransactionState> readyTransactionS
                     }
                     // clear publish version tasks to reduce memory usage when state changed to visible.
                     transactionState.clearPublishVersionTasks();
+
+                    // Refresh materialized view when base table update transaction has been visible
+                    long dbId = transactionState.getDbId();
```

Comment:
Because we only support total-refresh which is resource consuming, I'm afraid refreshing the MV when its base table is updated will bring our system a lot of load.
I think we need to give the MV a refresh strategy, we only refresh the auto-fresh MV automatically and not refresh other strategy such as manual-refresh.